### ISSUE TITLE
Trusted URI:s removing an item loses list focus

### ISSUE THREAD

**Issue by isidorn** [microsoft/vscode#127622]
Testing #127549

1. Keyboard focus on trusted URI list of items
2. Use tab to navigate to the X to remove an item. Press `space` to trigger removal
3. Item nicely gets removed from the list, however focus is lost 🐛 

Focus should be preserved in the list on the neighbour item (example: checkout how Explorer does that when deleting a file).
@joaomoreno I thought this would be tackled via https://github.com/microsoft/vscode/issues/126964

**Comment by sbatten** [comment]
The trust editor does a lot of full page re-rendering which I should improve (remove) so I doubt this is an issue with the table widget itself.
